lib/kb_mash/mash_utils/MashUtils.py

```diff
@@ -14,6 +17,20 @@ def log(message, prefix_newline=False):
     print(('\n' if prefix_newline else '') + '{0:.2f}'.format(time.time()) + ': ' + str(message))
 
 
+def logerr(err):
+    '''
+    Log an exception's traceback, split over several lines.
+    '''
+    s = traceback.format_exc(err).split('\n')  # I asssume there's a better way to do this
```

Comment:
I'd reformat this into the following to avoid single letter vars:
```py
for line in traceback.format_exc(err).split('\n'):
    log(line)
```

Reply:
done